### DISCUSSION TITLE
Status page using NRQL pulls all metrics and gives health of Application.

### DISCUSSION THREAD

**Discussion by Nchittoor** [newrelic/nr1-status-pages#159] [General]
Can you please help with an NRQL which Pulls health status on an application by pulling data from APM,INFRA,BROWESER,MOBILE and SYNTHETICS CHECKS in NR

**Comment by aso1124** [comment]
Hi @Nchittoor - just to confirm, are you looking to create a single status card to report this data, or split them across multiple cards?

**Comment by aso1124** [comment]
Hey @Nchittoor - I think I kind of misunderstood what you were getting at. You need an overall health status that reflects the underlying status of all the constituent parts of a business application, as reflected in those various event types you listed?

For that, I'd recommend you use a [Workload](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/workloads/workloads-isolate-resolve-incidents-faster/). Workloads allow you to create logical groupings of entities. [Entities](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) are the individual components that you monitor with New Relic - your server side applications (APM), browser and mobile apps, infra, synthetics, etc. 

In addition to providing preconfigured views scoped to its set of entities, a workload also reports an aggregate health status, derived from the individual health status of each entity. Entity health is based on its alert status; that is, whether or not it has active incidents open against it.

Once you have created the Workload, you can additionally configure a Workload card in Status Pages if needed. When adding a new service, select the Workload provider and enter the guid for your Workload (you can find the guid in the workload metadata - access that by clicking on the metadata bar under the Workload title, as you can see in this image).

![image](https://github.com/newrelic/nr1-status-pages/assets/1786630/b18f2a6e-6dab-40c2-81f5-7ec3d7b5724f)

Hope this helps!
